scenarios/v5/cluster/0_start.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e pipefail
+
+CDEMO_ROOT=../../..
+CONJUR_IMAGE=registry.tld/conjur-appliance:5.0-stable
+
+function main {
+  initialize
```

Comment:
What do you think about checking to see if any relevant containers are running prior to asking if we should remove containers?  Something like:
```sh
  if [ -n "$(docker-compose ps -q)" ]; then
    initialize
  fi
```